app/auth/templates/auth/login.html

```diff
@@ -49,8 +49,10 @@
     Don't have an account yet? <a href="{{ url_for('auth.register') }}">Sign up</a>
   </div>
 
+  {% if (GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET) or (GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET) or (FACEBOOK_CLIENT_ID and FACEBOOK_CLIENT_SECRET) %}
```

Comment:
to make these variables available in the Jinja2 templates, I think you'd need to include them here: https://github.com/simple-login/app/blob/5636b7ba32c84d0ff1a7cb1717664f684689a93c/server.py#L449

Reply:
Thanks for letting me know (I'm new to Jinja). I included them now. Is it possible to include all `env` variables by default? From my point of view this would make sense.